crates/noirc_evaluator/src/ssa/conditional.rs

```diff
@@ -454,7 +459,7 @@ impl DecisionTree {
             stack.push(ins2);
             true
         } else {
-            false
+            unreachable!("{}", error_msg);
```

Comment:
Since this function always returns true now, can we remove the return value and return `false` directly where it is used below?

Reply:
Done